experimental/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ All notable changes to experimental packages in this project will be documented
 
 ### :bug: (Bug Fix)
 
+* fix(instrumentation-http): drop url.parse in favor of URL constructor [#5091](https://github.com/open-telemetry/opentelemetry-js/pull/5091) @pichlermarc
+  * fixes a bug where using cyrillic characters in a client request string URL would throw
```

Comment:
Looks like there's some missing in this sentence. Assuming it was meant to be something along the lines of "would throw exceptions".

Reply:
thanks for the feedback - I tried to address this in https://github.com/open-telemetry/opentelemetry-js/pull/5091/commits/b0d1c8c0f7a21e23fc47ad817ac8068bd6513aa8. Please let me know if it's still unclear :slightly_smiling_face: